pkgs/top-level/python-packages.nix

```diff
@@ -105,6 +105,16 @@ in {
 
   aenum = callPackage ../development/python-modules/aenum { };
 
+  affinity = buildPythonPackage rec {
```

Comment:
See the header of this file and the Nixpkgs manual for contributing Python packages.